ui/src/sd-admin/studyAdmin.tsx

```diff
@@ -0,0 +1,497 @@
+                top: "-16px",
+              },
+            }}
+            name="irbNumber"
+            onChange={({ target: { name, value } }) => filterFn(name, value)}
+            label="Filter"
+            variant="standard"
+            size="small"
+            margin="none"
+          />
+        </div>
+      </div>
+    ),
+  },
+];
+
+// Temp PI options until endpoint in place
+const piOptions = ["", "Will", "Erik", "Tim", "Brian", "Chenchal"];
+
+const getValueFromStudyProperty = (
+  properties: StudyV2Property[],
+  key: string
+) => properties?.find((pair) => pair.key === key)?.value;
+
+const mapStudyRows = ({
+  id,
+  displayName,
+  description,
+  properties,
+}: StudyV2) => ({
+  id,
+  displayName,
+  description,
+  irbNumber: getValueFromStudyProperty(
+    properties as StudyV2Property[],
+    "irbNumber"
+  ),
+  pi: getValueFromStudyProperty(properties as StudyV2Property[], "pi"),
+});
+// Style override to slightly darken the text of disabled form fields
+const DISABLED_SX = [
+  {
+    ".Mui-disabled": {
+      color: "rgba(0, 0, 0, 0.6)",
+      "-webkit-text-fill-color": "rgba(0, 0, 0, 0.6)",
+    },
+  },
+];
+const ROWS_PER_PAGE = 25;
+
+const emptyStudy: StudyV2 = {
+  id: "",
+  displayName: "",
+  description: "",
+  properties: [
+    { key: "irbNumber", value: "" },
+    { key: "pi", value: "" },
+  ],
+};
+
+const initialFormState = {
+  displayName: {
+    touched: false,
+    value: "",
+  },
+  description: {
+    touched: false,
+    value: "",
+  },
+  irbNumber: {
+    touched: false,
+    value: "",
+  },
+  pi: {
+    touched: false,
+    value: "",
+  },
+  piInput: {
+    touched: false,
+    value: "",
+  },
+};
+
+const requiredFields = ["displayName", "irbNumber", "pi"];
+
+interface StudyV2Property {
+  key: string;
+  value: string;
+}
+
+interface StudyRow {
+  id: string;
+  displayName: string;
+  description: string;
+  irbNumber: string;
+  pi: string;
+}
+
+export function StudyAdmin() {
+  const source = useAdminSource();
+  const [activeStudy, setActiveStudy] = useState<StudyV2>(emptyStudy);
+  const [formState, setFormState] = useState(initialFormState);
+  const [columnFilters, setColumnFilters] = useState({
+    displayName: "",
+    irbNumber: "",
+  });
+  const [creatingStudy, setCreatingStudy] = useState<boolean>(false);
+  const [editingStudy, setEditingStudy] = useState<boolean>(false);
+  const [loadingStudy, setLoadingStudy] = useState<boolean>(false);
+  const [loadingStudyList, setLoadingStudyList] = useState<boolean>(true);
+  const [studies, setStudies] = useState<StudyV2[]>([]);
+
+  useEffect(() => {
+    getStudies();
+  }, []);
+
+  const getStudies = async () => {
+    const studies = await source.getStudiesList();
+    setStudies(studies);
+    setLoadingStudyList(false);
+  };
+
+  const updateStudy = async () => {
+    setLoadingStudy(true);
+    const updateStudyRequest: UpdateStudyRequest = {
```

Comment:
Optional: This is the sort of stuff I would put in AdminSource and why I created the Source in the first place, so that the UI doesn't have to know about the details of the API calls and would instead call simple functions like `updateStudyInfo(id, name, description)` or use objects (e.g. a Study) defined in a UI-centric manner.

Reply:
Ah, that makes sense. Updated the `createStudy` function also.